jwst/tweakreg/matchutils.py

```diff
@@ -1,3 +1,10 @@
+"""
+A module that provides algorithms for initial estimation of shifts
+based on 2D histograms.
+
+:License: `<http://www.stsci.edu/resources/software_hardware/pyraf/LICENSE>`_
```

Comment:
Is this the appropriate license reference to use? This isn't part of pyraf.

Reply:
@hbushouse Oh, this is used almost everywhere in my contributed code - I carried it over from HST module "model". I will see how I can make this to point to the `LICENSE` file in the root directory.